Clients/src/presentation/pages/SettingsPage/Organization/index.tsx

```
@@ -173,9 +293,11 @@ const Organization = () => {
         body: "The organization was updated successfully.",
         isToast: false,
       });
+
       if (response && response.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)